### PR TITLE
CLP-11811 copy cell properties from table styles

### PR DESCRIPTION
**Pull request by jurisdatum**
Sometimes there is cell formatting associated with a _table_ style. In the XML, we might see CSS in the `<presentation>` block that looks like this:
```CSS
.TableGrid td { border: 0.5pt solid; }
```
and then some XML in the body that looks like this:
```XML
<table class="TableGrid">
   <tr>
      <td>...</td>
   </tr>
</table>
```
With the included fix, these cell properties will be copied into the style attribute of the HTML <td> like this:
```HTML
<table>
   <tr>
      <td style="border: 0.5pt solid">...</td>
   </tr>
</table>
```
We already do something similar with inline styling contained in a paragraph style. This fix just extends it to table styles.
This fix is needed for [2024] UKFTT 564 (TC). See Jira issue CLP-11811

### CODE REVIEW

File: src/main/ml-modules/root/judgments/xslts/accessible-html.xsl [hunk 1]
```diff
@@ -331,10 +331,20 @@
...
+	<xsl:for-each select="key('classes', $key, $all-classes-parsed)/uk:property">
+		<xsl:sequence select="concat(@key, ':', @value)" />
+	</xsl:for-each>
+</xsl:function>
+
 <!-- returns a sequence of key/value pairs, each divided by a colon -->
 <xsl:function name="uk:get-inline-class-properties-1" as="xs:string*">
 	<xsl:param name="context" as="element()" />
-	<xsl:param name="selector" as="xs:string" />
+	<xsl:param name="selector" as="xs:string" /> <!-- constructed with helper function below -->
```
Comment (dragon-dxw) [inline src/main/ml-modules/root/judgments/xslts/accessible-html.xsl:347]: might be worth naming explicitly in case other functions get added.

**Review by dragon-dxw — APPROVED** [review]

File: src/main/ml-modules/root/judgments/xslts/accessible-html.xsl [hunk 2]
```diff
@@ -345,29 +355,40 @@
...
+		</xsl:otherwise>
+	</xsl:choose>
+</xsl:function>
+
+<xsl:function name="uk:make-class-selector" as="xs:string?">
+	<xsl:param name="context" as="element()" />
+	<xsl:param name="e" as="element()" />
+	<xsl:choose>
+		<xsl:when test="empty($e/@class)" />
+		<xsl:when test="$doc-id = 'ewhc/ch/2022/1178'">
```
Comment (dragon-dxw) [inline src/main/ml-modules/root/judgments/xslts/accessible-html.xsl:387]: Is it worth trying to rewrite that judgment semi-manually to try to make it comply with everything else?

**Review by dragon-dxw — APPROVED** [review]

File: src/main/ml-modules/root/judgments/xslts/accessible-html.xsl (hunk 1 above)
Comment (jurisdatum) [reply src/main/ml-modules/root/judgments/xslts/accessible-html.xsl]: ok, this is done

File: src/main/ml-modules/root/judgments/xslts/accessible-html.xsl (hunk 2 above)
Comment (jurisdatum) [reply src/main/ml-modules/root/judgments/xslts/accessible-html.xsl:387]: Not a bad idea. Let me think about it.

**Pull request merged**